models/yiiModels/RadiometricTargetSearch.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+//******************************************************************************
+//                                       RadiometricTargetSearch.php
+// SILEX-PHIS
+// Copyright © INRA 2018
+// Creation date: 27 September, 2018
+// Contact: vincent.migot@inra.fr, anne.tireau@inra.fr, pascal.neveu@inra.fr
+//******************************************************************************
+
+namespace app\models\yiiModels;
+
+/**
+ * implements the search action for the radioemtric target
```

Comment:
Please change "radioemtric" for "radiometric". 
Please uppercase the first word of the sentence